engine/src/main/java/org/terasology/physics/bullet/shapes/BulletCapsuleShape.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.physics.bullet.shapes;
+
+import com.badlogic.gdx.physics.bullet.collision.btCapsuleShape;
+import org.terasology.math.geom.Quat4f;
+import org.terasology.physics.shapes.CollisionShape;
+
+public class BulletCapsuleShape extends BulletCollisionShape {
```

Comment:
Ideally this should also implement an engine `CapsuleShape` interface, but it looks like I may have forgotten to add the interface to the CollisionShape API. I will add it in #3290, after which it can be implemented here.